lib/logstash/environment.rb

```diff
@@ -90,16 +90,17 @@ def test?
       env.downcase == "test"
     end
 
-    def bundler_setup!
+    def bundler_setup!(options = {})
+      options = {:without => [:development]}.merge(options)
+      options[:without] = Array(options[:without])
       # make sure we use our own nicely installed bundler and not a rogue, bad, mean, ugly, stupid other bundler. bad bundler, bad bad bundler go away.
       Gem.clear_paths
       Gem.paths = ENV['GEM_HOME'] = ENV['GEM_PATH'] = logstash_gem_home
       require "bundler"
       require "logstash/bundler"
 
-      ::Bundler.settings[:path] = LogStash::Environment::BUNDLE_DIR
-      ::Bundler.settings[:without] = "development"
-
+      ::Bundler.settings[:path]    = LogStash::Environment::BUNDLE_DIR
+      ::Bundler.settings[:without] =  options[:without].join(":")
```

Comment:
ok, sorry to be picky but you go extra length to align the `=` signs but you have an extra space in front of `options`